apps/sms/test/marionette/lib/views/conversation/message_accessors.js

```diff
@@ -52,6 +54,16 @@ MessageAccessor.prototype = {
     }
   },
 
+  selectMessage: function(messageNode) {
+    if (this.parse(messageNode).isSelected) {
+      throw new Error('Message has been selected!');
+    }
+    //messageNode.tap();
+    //messageNode.findElement('li[class=message]').tap();
+    //messageNode.findElement('.pack-checkbox').click();
+    messageNode.findElement('.pack-checkbox').tap();
```

Comment:
Well, you can't select first message because it's partially hidden by date header (when you call element.tap - it tries to tap in the middle of the element) :D So you have two choices, either select [1, 3] messages (2nd and 4th) instead of use ```scrollIntoView``` for the message node, take a look at how I modified your test a bit:
```
// I've moved it to ```view.js``` - note that we should avoid using ```accessor```
// in tests directly, only view should be able to access accessors
// (there is one temporary exception - ```accessors.waitToAppear```)
selectMessage: function(messageId) {
  var messageNode = this.messageAccessors.find(messageId);
  messageNode.scriptWith(function(node) {
    node.scrollIntoView();
  });
  messageNode.tap();
}

test.only('select only a few', function () {
  var messageIndicesToSelect = [0, 2];

  conversationView.enterEditMode();

  var messages = conversationView.messages;

  messageIndicesToSelect.forEach(function(messageIndex) {
    conversationView.selectMessage(messages[messageIndex].id);
  });

  conversationView.messages.forEach(function(message, index) {
    if (messageIndicesToSelect.indexOf(index) >= 0) {
      assert.isTrue(message.isSelected);
    } else {
      assert.isFalse(message.isSelected);
    }
  });
});
```